src-tauri/src/farmer.rs

```diff
@@ -170,7 +170,7 @@ async fn farm(base_directory: PathBuf, farm_args: FarmingArgs) -> Result<(), any
             enable_farming: true,
             relay_server_node,
         },
-        plot_size,
+        plot_size * 92 / 100, // LegacyMultiPlotsFarm was reducing input to 92% before, but it is not going it anymore. This is to make code equivalent to the old version.
```

Comment:
This in farmer much more readable and clearly states that this is not a final value:
https://github.com/subspace/subspace/blob/a1b9a3991a8db5d598f616d5c3f34cb8c54905ac/crates/subspace-farmer/src/bin/subspace-farmer/utils.rs#L28-L32